test/support/utils.h

```diff
@@ -930,6 +932,41 @@ get_new_kernel_params(KernelParams)
 }
 #endif //TEST_DPCPP_BACKEND_PRESENT
 
+template <typename T>
+typename ::std::enable_if_t<std::is_arithmetic_v<T>, void>
+generate_arithmetic_data(T* input, std::size_t size, std::uint32_t seed)
+{
+    std::default_random_engine gen{seed};
+    std::size_t unique_threshold = 75 * size / 100;
```

Comment:
I would add a comment why we use so "magic" constants (75 and 100)

Reply:
I've added a comment